egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -51,33 +76,34 @@
   word = line_vect[4]
   phones = line_vect[5]
   if uttID in utt_word_dict.keys():
-    utt_word_dict[uttID][count] = word
-    utt_phone_dict[uttID][count] = phones
+    utt_word_dict[uttID][utt_word_count] = word
+    utt_phone_dict[uttID][utt_word_count] = phones
   else:
-    count = 0
+    utt_word_count = 0
     utt_word_dict[uttID] = dict()
     utt_phone_dict[uttID] = dict()
```

Comment:
Actually I think it's more straightforward to use two lists rather than two dictionaries indexed by integers. Then you don't need the counter at all.